modelconverter/utils/config.py

```diff
@@ -134,41 +134,9 @@ class InputConfig(OutputConfig):
     ] = RandomCalibrationConfig()
     scale_values: Optional[Annotated[List[float], Field(min_length=1)]] = None
     mean_values: Optional[Annotated[List[float], Field(min_length=1)]] = None
-    reverse_input_channels: bool = False
```

Comment:
IMHO we should keep `reverse_input_channels` as a property of the `InputConfig` so the rest of the code doesn't have to be changed much. Checking `if inp.reverse_input_channels` is much nicer then always doing `if inp.encoding.to != inp.encoding.from_`.

Reply:
If you mean keeping `reverse_input_channels` as a flag in the config.yaml I don't agree with this. It just polutes the namescape and `encoding` block is much clearer IMO. But if you mean having it just as an internal property of the `InputConfig` then this sounds good. But in that case I would rename it to something like `same_input_output(self)->bool` (or maybe better name) which would just compare `to` and `from` encodings and check if there are the same. I would just stir away from the name `reverse_input_channels` as it is not descriptive enough.

Reply:
> If you mean keeping reverse_input_channels as a flag in the config.yaml I don't agree with this.
No, that's not what I mean. It would be purely internal and not exposed to the users.

Reply:
I think this is now the case with the [encoding_mismatch](https://github.com/luxonis/modelconverter/blob/50f4968a09014c8816b0d7a32e64fc6796fbc511/modelconverter/utils/config.py#L140-L142) property.